docs/ReleaseNotes.Xerus.October.2018.md

```diff
@@ -31,7 +31,7 @@ Kafka queue retention time enlarged to 1 week | [895](https://github.com/thomson
 DCC false positive for null updates | [883](https://github.com/thomsonreuters/CM-Well/pull/883) | DCC would falsely identify an infoton as changed when in fact it was a null update (an update where no field values have changed).
 DCC false positive if CAS version updated before ES | [888](https://github.com/thomsonreuters/CM-Well/pull/888) | If the DCC tested a new infoton that was updated in Cassandra but not in ElasticSearch yet, it would identify a false positive.
 DCC false positive for deleted infotons | [900](https://github.com/thomsonreuters/CM-Well/pull/900) | Deletion of a non-existent infoton caused a false positive.
-DCC false positive for 1ms difference in update time | [908](https://github.com/thomsonreuters/CM-Well/pull/908), [910](https://github.com/thomsonreuters/CM-Well/pull/910) | In some cases, infoton update times in ES and CAS have a difference of 1ms. This is caused by an "artificial" update time that solves a synchronization problem. This case was previously identified as a data inconsistency, but should not be.
+DCC false positive for 1ms difference in update time | [908](https://github.com/thomsonreuters/CM-Well/pull/908), [910](https://github.com/thomsonreuters/CM-Well/pull/910) | In some cases, infoton update times in ES and CAS have a difference of 1ms. This is caused by an "artificial" update time that addresses the problem of two writes of the same path having reverse order in the Kafka queue. This case was previously identified as a data inconsistency, but should not be.
```

Comment:
There shouldn't be a 1ms diff. In order to solve the reverse order issue BG adds 1ms to the time written by the WS and writes it to CAS and to ES.

Reply:
Amended according to your comment; please see the updated file.